.github/workflows/ci.yml

```diff
@@ -7,21 +7,21 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_master.gemfile]
-        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
+        gemfile: [rails_4.0.0.gemfile, rails_5.0.0.gemfile, rails_5.2.3.gemfile, rails_6.0.0.gemfile, rails_7.0.0.gemfile, rails_master.gemfile]
+        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x, 3.0.x]
         exclude:
           - gemfile: rails_master.gemfile
-            ruby_version: 2.4.x
-          - gemfile: rails_6.0.0.gemfile
-            ruby_version: 2.4.x
+            ruby_version: 2.7.x
+          - gemfile: rails_7.0.0.gemfile
+            ruby_version: 2.7.x
```

Comment:
why do we need to exclude `rails_master + ruby 2.7.x` and `rails_7 + ruby 2.7.x` ?

Reply:
rails 7 support only ruby 2.7+. That's why I added a the new ruby version. I'm not sure how to make this work with ruby 2.7 in the GitHub actions pipeline the way it's setup though.

Reply:
![image](https://user-images.githubusercontent.com/6373536/147761345-486df854-9884-4575-8f9b-242f86656b3c.png)

Right, it's fine to add the new gemfile and ruby version, the problem is that you are `excluding` them as you can see in the image

Reply:
I think what we need to do is only exclude unsupported combinations. For example Rails 7 doesn't work with Ruby <= 2.6. Rails 6 doesn't work with Ruby <= 2.4, so those cases needs to be added to the `exclude`

Reply:
oh, I see, thanks for the explanation